notebooks/community/model_evaluation/automl_tabular_classification_model_evaluation.ipynb

```diff
@@ -0,0 +1,1375 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #19.        dataflow_machine_type: str = 'n1-standard-4',
</pre><p>Please remove all dataflow params that we're not modifying and have default values. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/911/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='58'/>

Reply:
<p>the argument descriptions in the <a href="https://google-cloud-pipeline-components.readthedocs.io/en/google-cloud-pipeline-components-1.0.17/google_cloud_pipeline_components.experimental.evaluation.html#google_cloud_pipeline_components.experimental.evaluation.ModelEvaluationFeatureAttributionOp" target="_blank">docs</a> suggest different default values than what are being set against the argument names in the definition. For ex., <strong><em>dataflow_max_workers_num </em></strong>is defaulted to 5 in the definition but the description says it'd be defaulted to 25.</p>

Reply:
<p>done.</p>